addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -93,38 +92,30 @@ export class ControlPanel extends Component {
          * and the currrent userId. Each key contains a dict. The keys of the latter are the id of the visible embedded
          * actions.
          */
-        const parentActionId =
+        this.parentActionId =
             this.env.config.parentActionId ||
             this.env.config.embeddedActions?.[0]?.parent_action_id[0] ||
             this.env.config.embeddedActions?.[0]?.parent_action_id ||
             "";
-        this.embeddedActionsVisibilityKey = `visibleEmbeddedActions${parentActionId}+${
+        this.resId = this.env.searchModel?.globalContext.active_id;
```

Comment:
I'd name resId as currentActiveId, it's always a benefit to keep the semantics of a concept: activeId depends on the context, but resId depends on the associated resModel that goes mandatorily with it